configure.ac

```diff
@@ -134,11 +134,13 @@ AS_IF([test x"$with_rpc" != "xno"], [
         have_rpc=maybe
         save_CPPFLAGS=$CPPFLAGS
         CPPFLAGS="$CPPFLAGS $RPC_CFLAGS"
-        AC_CHECK_HEADERS([rpc/rpc.h],, [have_rpc=no])
+        dnl XXX The fourth argument to AC_CHECK_HEADERS is required for autoconf
```

Comment:
use `TODO` instead of `XXX`.  the `XXX` marker is ambiguous as to its meaning -- is it a note?  is it a todo?  should someone periodically review this ?

Reply:
Fixed.